docs/take-to-prod.md

```diff
@@ -0,0 +1,108 @@
+Please see below as an example of running your app with `spark-submit` in Scenario 1 and Scenario 2.
+```shell
+%SPARK_HOME%\bin\spark-submit \
+--class org.apache.spark.deploy.dotnet.DotnetRunner \
+--master local \
+--files bin\Debug\netcoreapp3.0\mySparkApp.dll \
+bin\Debug\<dotnet version>\microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar \
+dotnet bin\Debug\netcoreapp3.0\mySparkApp.dll <app arg 1> <app arg 2> ... <app arg n>
+```
+#### 2. Using Apache Livy
+Please see below as an example of running your app with Apache Livy in Scenario 1 and Scenario 2.
+```shell
+{
+    "file": "adl://<cluster name>.azuredatalakestore.net/<some dir>/microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar",
+    "className": "org.apache.spark.deploy.dotnet.DotnetRunner",
+    "files": [“adl://<cluster name>.azuredatalakestore.net/<some dir>/mySparkApp.dll" ],
+    "args": ["dotnet","adl://<cluster name>.azuredatalakestore.net/<some dir>/mySparkApp.dll","<app arg 1>","<app arg 2>,"...","<app arg n>"]
+}
+```
+
+## How to deploy your application when you have multiple dependencies
+### Scenarios - multiple dependencies
+#### Scenario 3. SparkSession code in one project that references another project including the business logic
+This would be the use case when you have `SparkSession` code in one project (e.g. mySparkApp.csproj) and business logic (UDFs) in another project (e.g. businessLogic.csproj).
+#### Scenario 4. SparkSession code references a function from a Nuget package that has been installed in the csproj
+This would be the use case when `SparkSession` code references a function from a Nuget package in the same project (e.g. mySparkApp.csproj).
+#### Scenario 5. SparkSession code references a function from a DLL on the user's machine
+This would be the use case when `SparkSession` code reference business logic (UDFs) on the user's machine (e.g. `SparkSession` code in the mySparkApp.csproj and businessLogic.dll on a different machine). 
+#### Scenario 6. SparkSession code references functions and business logic from multiple projects/solutions that themselves depend on multiple Nuget packages
+This would be a more complex use case when you have `SparkSession` code reference business logic (UDFs) and functions from Nuget packages in multiple projects and/or solutions.
+
+### Package your application - multiple dependencies
+- Please follow [Get Started](https://github.com/dotnet/spark/#get-started) to build your mySparkApp.csproj in Scenario 4 and Scenario 5 (and businessLogic.csproj for Scenario 3).
+- Please see detailed steps [here](https://github.com/dotnet/spark/tree/master/deployment#preparing-your-spark-net-app) on how to build, publish and zip your application in Scenario 6. After packaging your .NET for Apache Spark application, you will have a zip file (e.g. mySparkApp.zip) which has all the dependencies.
+
+### Launch your application - multiple dependencies
```

Comment:
remove `-multiple dependencies`

Reply:
So all the suffix of `--single dependency` and `multiple dependencies` are for `table of content`. Since we have `Scenarios`, `Package your application`, `Launch your application` in both single dependency and multiple dependencies, I want to use these suffix to different them in these two sections for the linking of `table of content`, otherwise they will be all link to the first section.

Reply:
Please see this [example](https://github.com/dotnet/spark/tree/master/deployment).